index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+ <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/53a64647bf.js" crossorigin="anonymous"></script>
+    <title>Portfolio-website</title>
+ </head>
+
+ <body>
+   <header>
+       <nav class="navbar">
+           <a href="#">LOGO</a>         
```

Comment:
- [ ] Kindly use the same font color used in the figma design for your logo, please check screenshot below  👍🏿